packages/requisitions/src/CustomerRequisition/DetailView/Footer.tsx

```diff
@@ -9,11 +9,11 @@ import {
 } from '@openmsupply-client/common';
 import React, { FC } from 'react';
 import {
-  getSupplierRequisitionStatuses,
-  getNextSupplierRequisitionStatus,
-  getSupplierRequisitionTranslator,
+  getRequestRequisitionStatuses,
```

Comment:
just curious / confused. Is this due to the reuse of status across both request and response requisitions?
We're in the `CustomerRequisition` directory using the `RequestRequisitionStatus`

Reply:
Oh, I see that there's a `RequisitionNodeStatus` later on.. maybe copy paste from supplier reqs then?

Reply:
Sorry - have made this for spending some more time on statuses https://github.com/openmsupply/openmsupply-client/issues/849